app/controllers/editor/pickup_posts_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Editor::PickupPostsController < Editor::ApplicationController
+  def index
+    @pickup_posts = pickup_posts.includes(:post).ordered.page(params[:page]).per(200)
+  end
+
+  def new
+    @pickup_post = pickup_posts.build
+  end
+
+  def create
+    @pickup_post = pickup_posts.build(pickup_post_params)
+
+    if @pickup_post.save
+      redirect_to [:editor, :pickup_posts], notice: "ピックアップ記事の設定が作成されました"
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @pickup_post = pickup_posts.find(params[:id])
+  end
+
+  def update
+    @pickup_post = pickup_posts.find(params[:id])
+
+    if @pickup_post.update(pickup_post_params)
+      redirect_to [:editor, :pickup_posts], notice: "ピックアップ記事の設定が更新されました"
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @pickup_post = pickup_posts.find(params[:id])
```

Comment:
How about using local var (`pickup_post`) instead of instance var (`@pickup_post`) ?
It seems that we don't have any view for `destroy`.